frontend-react/src/hooks/UseLookupTable.ts

```diff
@@ -139,4 +135,13 @@ const useLookupTable = <T>(
     return valueSetArray;
 };
 
+/* Keeping the dataSetName in here would keep this from refreshing if it changes. */
+export const useValueSetsTable = (dataSetName: string | null) =>
```

Comment:
since we don't need dataSetName for this use case, I think this function can safely just take no arguments. Would need to adjust the call in ValueSetsIndex as well.

Reply:
Fixed this up and removed the unnecessary comment above it that I'd leftover from when it had a default string lol